ReleaseNotes.md

```diff
@@ -9,8 +9,8 @@ additional questions or comments.
 
 ## Release History
 
-### 11/29/2023 -
-##### Version microsoft/oms:3.1.15 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.16 (linux)
+### 12/08/2023 -
+##### Version microsoft/oms:3.1.16 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.16 (linux)
```

Comment:
we dont use microsoft/oms:3.1.16 and microsoft/oms:3.1.15 so these can be removed.